src/Viewer.tsx

```diff
@@ -449,8 +450,19 @@ function Viewer(): ReactElement {
           datasetKey = newCollection.getDefaultDatasetKey();
         } else {
           // Try loading the collection, with the default collection as a fallback.
-          newCollection = await Collection.loadCollection(collectionUrlParam || DEFAULT_COLLECTION_PATH);
-          datasetKey = datasetParam || newCollection.getDefaultDatasetKey();
+          try {
+            newCollection = await Collection.loadCollection(collectionUrlParam || DEFAULT_COLLECTION_PATH);
+            datasetKey = datasetParam || newCollection.getDefaultDatasetKey();
+          } catch (error) {
+            console.error(error);
+            notificationApi["error"]({
+              message: "Error loading dataset:",
```

Comment:
does the url of the dataset come in here via the error? Would it be good to add collectionUrlParam to the message explicitly here?

Reply:
It usually does, though it'll depend on the error message. The console log statement will have it though.
![image](https://github.com/allen-cell-animated/nucmorph-colorizer/assets/30200665/a1134766-d18e-447c-9cd2-3beec19be2e5)

Reply:
![image](https://github.com/allen-cell-animated/nucmorph-colorizer/assets/30200665/c65e0ab7-caa3-41f6-83cc-7d04be32948e)